doc/subsystems/logging/logger.rst

```diff
@@ -221,6 +228,13 @@ be used to change maximal severity level for given module. Module is identified
 by source ID and domain ID. Source ID can be retrieved if source name is known
 by iterating through all registered sources.
 
+If logger is processed from a thread then it is possible to enable a feature
+which will wake up processing thread when certain amount of log messages are
+buffered (see :option:`CONFIG_LOG_PROCESS_TRIGGER_THRESHOLD`). It is also
+possible to enable internal logger thread (see
+:option:`CONFIG_LOG_PROCESS_THREAD`). In that case logger is initialized and
```

Comment:
Last sentence is unclear to me. 
What about something like this: "In that case logger thread is initialized and log messages are processed implicitly."?